jsonic3_commit.py

```diff
@@ -0,0 +1,80 @@
+"""
+Grabs data from the 'FAQ Content' CSV and turns it into nice JSON: a main faq object containing an array of Section objects, each Section Object in turn holding an array of Question objects consisting of question, answer and related link/s, as so:
+    [
+        {
+          'Q': 'What are the symptoms of COVID-19?',
+          'A': 'Symptoms of COVID-19 include coughing and shortness of breath. Additionally, a person showing two or more of the following symptoms may have the virus: fever, repeated shaking with chills, muscle pain, headache, sore throat, new loss of taste or smell.',
+          'link': 'https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html'
+        },
+    .
+    .
+    .
+    ]
+    etc
+In addition to an array of Question objects, each Section incorporates its title and a 'last updated' date value. For the Python intermediary stage (between CSV stream  and JSON) we will create a hierarchy of nested dictionaries and lists equivalent to the JSON objects and arrays.
+"""
+
+import requests
+import json
+import csv
+import datetime
+from typing import Iterable, Dict, List, Any
+
+
+# function to grab data from a google sheet + return as reader object
+def google_sheet_csv_data(sheet: str, gid: str) -> Iterable[List[str]]:
+    url = f'https://docs.google.com/spreadsheets/d/{sheet}/export'
+    response = requests.get(url, params={
+        'format': 'csv',
+        'id': sheet,
+        'gid': gid
+    })
+    response.raise_for_status()
+    return csv.reader(response.iter_lines(decode_unicode=True))
+
+
+# Pass faq Content sheet id and gid to function
+_reader = google_sheet_csv_data('1_wBXS62S5oBQrwetGc8_-dFvDjEmNqzqHwUeP-DzkYs',
+ '1318925039')
+
+# Date, to be updated each time the program runs, in UTC
+date = datetime.datetime.utcnow().strftime('%Y-%m-%d')
```

Comment:
I don’t remember if I remarked on this before, but would it be good to include the time here so it’s clear when updates were if we ultimately run them more than once a day?

If the front-end will choke if time is included, then obviously disregard that entirely. 😅

Reply:
Actually, not too sure about this one, as the faqs are written by members of the team rather than scraped from somewhere else, so the likelihood of ever needing to run this twice a day automatically is pretty remote. If some piece of information changed and needed to be updated urgently, we'd probably just run the script manually. 
Also think Kengo will have kittens if we suggest adding time! :-)